app/src/main/java/com/github/fribourgsdp/radio/data/view/UserProfileActivity.kt

```diff
@@ -119,7 +119,10 @@ open class UserProfileActivity : MyAppCompatActivity(), KeepOrDismissPlaylistDia
         user.name = usernameField.text.toString()
         //at this point, the userId should be the firebaseUser.uid
         if(user.isGoogleUser){
-            db.setUser(user.id,user)
+            //remove all n
```

Comment:
What is `n`?

Reply:
I don't remember but I also removed this comment